cmd/esbulk/esbulk.go

```diff
@@ -42,6 +44,24 @@ var (
 	pipeline        = flag.String("p", "", "pipeline to use to preprocess documents")
 )
 
+//isJSON checks if a string is valid json
+func isJSON(s string) bool {
```

Comment:
There is a shorter way I believe, e.g. as shown here: https://stackoverflow.com/a/36922225/89391